app/routes/faqs.js

```diff
@@ -0,0 +1,5 @@
+import Ember from 'ember';
+import UnauthenticatedRouteMixin from 'ember-simple-auth/mixins/unauthenticated-route-mixin';
+
+export default Ember.Route.extend(UnauthenticatedRouteMixin, {
```

Comment:
I think you need to remove the UnauthenticatedRouteMixin.
This only lets unauthenticated users see this page:
https://ember-simple-auth.com/api/classes/UnauthenticatedRouteMixin.html